src/binder/cdebuglog.cpp

```diff
@@ -65,8 +65,11 @@ namespace BINDER_SPACE
                 IF_FAIL_GO(HRESULT_FROM_WIN32(ERROR_BUFFER_OVERFLOW));
             }
         
-            IF_FAIL_GO(StringCbCopy(szPath, sizeof(szPath), pszName));
-
+            WCHAR * szPath = szPathString.OpenUnicodeBuffer(MAX_LONGPATH);
```

Comment:
The point of using variable size optimized buffer class is that we should be allocating only what is needed whenever possible. We know exactly on how much we need to allocate here - so we should only allocate as much.